### ISSUE TITLE
Microservices, Docker, and Kubernetes

### ISSUE THREAD

**Issue by csharpfritz** [csharpfritz/csharp_with_csharpfritz#65]

**Comment by Mrxx99** [comment]
Looking forward for Tye, will you also cover Microsoft Orleans and Dapr?

**Comment by VincentVerweij** [comment]
Hi Jeff, wasn't this supposed to be shown in the season 2? Just saw that today you started the 3rd which is blazor.

**Comment by csharpfritz** [comment]
Yes. I decided to move into Blazor as there was interest in that and there are a few events coming up in the next week or two that will cover this better than I would so I thought it better to defer to them. 



Jeff
